src/components/Footer/Footer.Constants.ts

```diff
@@ -0,0 +1 @@
+export const possibleStatus = ['All', 'Active', 'Completed'];
```

Comment:
you don't need this array, you can get array from enum `Sort` in places where you need array with values